Before: df2ca9402ae08e52322d6f8e221e8ac381c9afa7eae6c91894fa6173a8f44020
After: 817e49280c3b0702bf116e181955be362bcc4472c8ab0f02d680b375e86802e0
Patch: b84774eea62b810b9d41bd71081cc2854c9cd00d8613426e39450ecc266d16fa
--- corvid-features/plugin.rb	2012-08-09 12:47:53.088740480 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,8 +0,0 @@
-since_ver 2
-
-install {
-  empty_directory 'resources'
-  empty_directory 'resources/latest'
-  copy_file       'resources/.gitignore'
-  install_feature 'test_spec', run_bundle: true, say_if_installed: false
-}
--- resources/.gitignore	2012-08-09 12:47:19.975219366 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1 +0,0 @@
-latest
